spec/requests/proceeding_types/threshold_waivers_controller_swagger_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "swagger_helper"
+
+RSpec.describe "threshold_waivers", type: :request do
```

Comment:
not sure but relative to the other threshold waivers spec perhaps
```suggestion
RSpec.describe "proceeding_types/threshold_waivers", type: :request do
```